test/Prototypes/string_prototype/main.swift

```diff
@@ -335,4 +335,15 @@ testSuite.test("substring") {
   expectEqualSequence("ello world", tail)
 }
 
+import Foundation
+testSuite.test("bridging") {
+  defer { _debugLogging = false }
+  _debugLogging = false
+  let s : main.String
```

Comment:
No, the intention was to change line 341 to `= true` to turn on logging for just this case.